rmgpy/rmg/main.py

```diff
@@ -764,15 +798,15 @@ def execute(self, **kwargs):
         # Run sensitivity analysis post-model generation if sensitivity analysis is on
         for index, reactionSystem in enumerate(self.reactionSystems):
             
-            if reactionSystem.sensitiveSpecies:
+            if reactionSystem.sensitiveSpecies and reactionSystem.sensConditions:
```

Comment:
I think that `sensConditions` is `None` if ranges aren't specified. Does this condition allow "normal" SA to be run?

Reply:
It should do so now since the non-ranged values will default as sensConditions